src/java/org/apache/cassandra/db/compaction/UnifiedCompactionStrategy.java

```diff
@@ -680,6 +777,22 @@ public Controller getController()
         return tables.values();
     }
 
+    // used by CNDB to deser aggregates
```

Comment:
I don't know such word `deser`. Is it a typo or is it a word, which is not known by me, non-native speaking? It would be nice to replace it with more familiar word.

Reply:
In this context, believe it's short for "deserialize" (otherwise I don't know either). It will be more clear to use the full word.